tests/PHPUnit/Unit/ConfigTest.php

```diff
@@ -390,5 +390,44 @@ public function test_forceSave_writesNothingIfThereAreNoChanges()
 
         @unlink($configFile);
     }
+    
+    public function testFromGlobalConfig()
+    {
+        $userFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/config.ini.php';
+        $globalFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/global.ini.php';
+        $commonFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/common.config.ini.php';
+        
+        $config = new Config($globalFile, $userFile, $commonFile);
+        $config->reload();
+        
+        $configCategory = $config->getFromGlobalConfig('Category');
+        $this->assertEquals('value1', $configCategory['key1']);
+    }
+    
+    public function testFromCommonConfig()
+    {
+        $userFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/config.ini.php';
+        $globalFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/global.ini.php';
+        $commonFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/common.config.ini.php';
+    
+        $config = new Config($globalFile, $userFile, $commonFile);
+        $config->reload();
+    
+        $configCategory = $config->getFromCommonConfig('Category');
+        $this->assertEquals('valueCommon', $configCategory['key2']);
```

Comment:
@diosmosis i added the test.

But as u can see in the previous commit, this test would fail:
```php
        $this->assertEquals('${@piwik(crash))}', $configCategory['key3']);
```